node/src/ui_gateway/websocket_supervisor.rs

```diff
@@ -133,22 +135,40 @@ impl WebSocketSupervisorReal {
             .collect()
     }
 
-    fn send_msg(locked_inner: &mut MutexGuard<WebSocketSupervisorInner>, msg: NodeToUiMessage) {
+    fn send_msg(inner_arc: &Arc<Mutex<WebSocketSupervisorInner>>, msg: NodeToUiMessage) {
+        let mut locked_inner = inner_arc.lock().expect("WebSocketSupervisor is poisoned");
         let clients = match msg.target {
             MessageTarget::ClientId(n) => {
-                let clients = Self::filter_clients(locked_inner, |(id, _)| **id == n);
+                let clients = Self::filter_clients(&mut locked_inner, |(id, _)| **id == n);
                 if !clients.is_empty() {
                     clients
                 } else {
                     Self::log_absent_client(n);
                     return;
                 }
             }
-            MessageTarget::AllExcept(n) => Self::filter_clients(locked_inner, |(id, _)| **id != n),
-            MessageTarget::AllClients => Self::filter_clients(locked_inner, |_| true),
+            MessageTarget::AllExcept(n) => {
+                Self::filter_clients(&mut locked_inner, |(id, _)| **id != n)
+            }
+            MessageTarget::AllClients => Self::filter_clients(&mut locked_inner, |_| true),
         };
         let json = UiTrafficConverter::new_marshal(msg.body);
-        Self::send_to_clients(clients, json);
+        if let Some(errors) = Self::send_to_clients(clients, json) {
+            drop(locked_inner);
+            Self::stc_errors(errors, inner_arc)
+        }
+    }
+
+    fn stc_errors(
+        errors: Vec<SendToClientWebsocketError>,
+        inner_arc: &Arc<Mutex<WebSocketSupervisorInner>>,
+    ) {
+        errors.into_iter().for_each(|e| match e {
+            SendToClientWebsocketError::FlushError((client_id, e)) => {
+                Self::handle_flush_error(e, inner_arc, client_id)
+            }
+            SendToClientWebsocketError::SendError(_) => (),
```

Comment:
Put in a comment here explaining why you're throwing away this error, even if it's a plaintive, helpless comment, like "I don't know what to do with this!"